src/app/public/pages/login/login.component.ts

```diff
@@ -0,0 +1,77 @@
+import {Component, inject, OnInit} from '@angular/core';
+import {FormControl, FormGroup, Validators} from "@angular/forms";
+import {User} from "../../../shared/interfaces/user.interfaces";
+import {AuthService} from "../../../shared/providers/services/auth.service";
+import {ActivatedRoute, Params, Router} from "@angular/router";
+
+@Component({
+  selector: 'app-login-page',
+  templateUrl: './login.component.html',
+  styleUrls: ['./login.component.scss']
+})
+export class LoginComponent implements  OnInit {
+
+  form = new FormGroup({
+    email: new FormControl('',
+      [
+        Validators.required,
+        Validators.email,
+      ]),
+    password: new FormControl('',
+      [
+        Validators.required,
+        Validators.minLength(6),
+        Validators.pattern('^(?=.*[a-z])(?=.*[A-Z])(?=.*\\d)[a-zA-Z\\d]{6,}$'),
```

Comment:
when logging in, these two validators above are not needed (Validators.minLength, Validators.pattern)